tests/test_v2.py

```diff
@@ -22,3 +22,16 @@ def test_v2_functional_time_value_zero(tmp_path):
 
     assert local_run.main(tmp_path, ["run_all"])
     assert (tmp_path / "output/id/report.html").exists()
+
+
+def test_v2_functional_time_ever(tmp_path):
+    analysis = v2.Analysis(**v2.TEST_DEFAULTS)
+    analysis.time_ever = True
```

Comment:
This test fails if `analysis.time_scale = None` and from Tom's [comment](https://bennettoxford.slack.com/archives/C0441RRL3RC/p1682329713905699?thread_ts=1682327101.918099&cid=C0441RRL3RC), I think this will happen if `time_ever` is True

Reply:
Have set to None by default